internal/write.go

```diff
@@ -36,6 +38,44 @@ type WriteParams struct {
 	Precision  api.WritePrecision
 }
 
+func (p *WriteParams) Flags() []cli.Flag {
```

Comment:
Especially here, I'd prefer to have all the `cli.Flag` definitions in once place instead of splitting them between `cmd/` and `internal/`

Reply:
Done – I think the improvements are 👍🏻